src/views/Home.vue

```diff
@@ -529,6 +529,17 @@ const RESOURCES = [
 
 /* Conferences */
 const CONFERENCES = [
+  {
+    "title": "ScapyCon Automotive",
+    "venue": "ScapyCon 2024",
+    "links": [
+      {
+        tooltip: "Website",
+        icon: mdiWeb,
+        link: "https://scapycon.com"
```

Comment:
Is the website live?

Reply:
This is just the redirect... I need to remove the "s"

Reply:
I fixed it